core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlan.java

```diff
@@ -259,11 +262,9 @@ public Plan buildQueryPlan() throws PlanGenException {
         buildOperators();
         validateOperatorGraph();
         connectOperators(operatorObjectMap);
+        HashMap<String, ISink> sinkMap = findSinkOperators(operatorObjectMap);
 
-        ISink sink = findSinkOperator(operatorObjectMap);
-        
-        Plan queryPlan = new Plan(sink);
-        return queryPlan;
+        return new MultipleSinkPlan(sinkMap);
```

Comment:
Since changes are made here, `Plan` class not longer has any instances. Refer to comments in `MultipleSinkPlan.java`.